system-addon/test/functional/mochitest/head.js

```diff
@@ -115,13 +113,9 @@ async function check_highlights_elements(selector, length, message) { // eslint-
   let browser = gBrowser.selectedBrowser;
   await waitForPreloaded(browser);
 
-  Services.prefs.setBoolPref("browser.newtabpage.activity-stream.feeds.section.highlights", false);
-  Services.prefs.setBoolPref("browser.newtabpage.activity-stream.feeds.section.highlights", true);
-
   await BrowserTestUtils.waitForCondition(() => content.document.querySelector(selector), `No elements matching ${selector} were found`);
```

Comment:
This `content.` use is from main and triggering CPOWs. And you need to update this everywhere. This wait aross the different "check"s (probably best to rename those to "test" as it's not a simple check) seems like it could be part of some helper that does waitForCondition as pat of `before`?